### PR TITLE
Saleor 1861 - add error handling to returns form

### PR DESCRIPTION
**Pull request by mmarkusik**
I want to merge this change because it adds error handling to returns page

**PR intended to be tested with API branch:** 

### Pull Request Checklist

1. [ ] All visible strings are translated with proper context.
1. [ ] All data-formatting is locale-aware (dates, numbers, and so on).
1. [ ] Translated strings are extracted.
1. [ ] Number of API calls is optimized.
1. [ ] The changes are tested.
1. [ ] Data-test are added for new elements.
1. [ ] Type definitions are up to date.
1. [ ] Changes are mentioned in the changelog.

### Test environment config

API_URI=https://master.staging.saleor.rocks/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-1861 Add error handling to fulfilment lines](https://app.clickup.com/t/2549495/SALEOR-1861)

File: src/orders/components/OrderReturnPage/OrderReturnRefundItemsCard/ProductErrorCell.tsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+import { defineMessages } from "react-intl";
+import { useIntl } from "react-intl";
+
+const useStyles = makeStyles(
+  theme => ({
+    container: {
+      position: "relative"
+    },
+    errorBox: {
+      backgroundColor: "#FE6D76",
```
Comment (dominik-zeglen) [inline src/orders/components/OrderReturnPage/OrderReturnRefundItemsCard/ProductErrorCell.tsx:13]: Shouldn't it come from `theme.palette`?

File: src/orders/components/OrderReturnPage/OrderReturnRefundItemsCard/ProductErrorCell.tsx [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+      borderRadius: 8,
+      padding: `${theme.spacing(2)}px ${theme.spacing(3)}px`,
```
Comment (dominik-zeglen) [inline src/orders/components/OrderReturnPage/OrderReturnRefundItemsCard/ProductErrorCell.tsx:15]:
```suggestion
      padding: theme.spacing(2, 3),
```

File: src/orders/components/OrderReturnPage/OrderReturnRefundItemsCard/ProductErrorCell.tsx [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+      position: "absolute",
```
Comment (dominik-zeglen) [inline src/orders/components/OrderReturnPage/OrderReturnRefundItemsCard/ProductErrorCell.tsx:16]: If I were you I'd use `Popper`

**Review by dominik-zeglen — CHANGES_REQUESTED** [review]

**Review by jwm0 — APPROVED** [review]
Apart from what @dominik-zeglen mentioned it looks good 👍

**Review by dominik-zeglen — APPROVED** [review]

File: src/orders/components/OrderReturnPage/OrderReturnRefundItemsCard/ProductErrorCell.tsx [hunk 4]
```diff
@@ -0,0 +1,98 @@
+import { makeStyles, TableCell, Typography } from "@material-ui/core";
```
Comment (dominik-zeglen) [inline src/orders/components/OrderReturnPage/OrderReturnRefundItemsCard/ProductErrorCell.tsx:1]: This could use some deep importing

**Review by dominik-zeglen — CHANGES_REQUESTED** [review]

**Pull request merged**